packages/edit-post/src/editor.native.js

```diff
@@ -173,8 +193,10 @@ export default compose( [
 	} ),
 	withDispatch( ( dispatch ) => {
 		const { switchEditorMode } = dispatch( editPostStore );
+		const { editEntityRecord } = dispatch( 'core' );
```

Comment:
I think using the store name directly is not preferred anymore (see [this PR that recently refactored lots of code to switch to using the exposed name](https://github.com/WordPress/gutenberg/pull/27414) instead). Let's follow the same pattern as the `editPostStore` above, thanks!

Reply:
Ah, thank you for raising that and the additional context, I've gone ahead to update it. 🙇‍♀️